client/src/App.js

```diff
@@ -5,6 +5,7 @@ import { BrowserRouter, Route, Switch } from 'react-router-dom';
 import { theme } from './themes/theme';
 import Login from './pages/Login';
 import SignUp from './pages/SignUp';
+import OnBoard from './pages/OnBoard';
```

Comment:
Have you added Prettier yet? When I pull your code to make local revisions, my commit history shows changes across every React file.

Reply:
That is strange .. will reinstall VSCode with all plugins, to be sure it works correctly.